### DISCUSSION TITLE
which one should I use

### DISCUSSION THREAD

**Discussion by MohammadZohar** [spaceuptech/space-cloud#1622] [Q&A]
so we're are developing a mobile app in flutter at first we considered firebase as our backend
but we want our data to be owned by us and served from our own servers in AWS or local servers
so we are looking for well documented and supported service to be our backend 
is space cloud a good place to go
and some tips if we were to continue with space cloud

**Comment by YourTechBud** [comment]
I suggest you consider using alternatives like supabase or hasura for new projects
